tox.ini

```diff
@@ -288,7 +289,8 @@ deps =
     datastore_motor-motorlatest: tornado
     datastore_pymongo-pymongo03: pymongo<4.0
     datastore_pymongo-pymongo04: pymongo<5.0
-    datastore_pymssql: pymssql
+    datastore_pymssql-pymssqllatest: pymssql
+    datastore_pymssql-pymssql020301: pymssql==2.3.1
```

Comment:
From the release notes, it seems to indicate that support for Python 3.7 and 3.8 was dropped in this version: https://pymssql.readthedocs.io/en/latest/changelog.html#version-2-3-1-2024-08-25-mikhail-terekhov

I was going to suggest pinning this to the previous version instead of this one.  However, if this works, we can keep it as is

Reply:
They dropped MacOS wheels for Python 3.7 & 3.8 in that version, they dropped the linux ones in the next version.